### PR TITLE
Bumping artifact versions for v1

### PR DESCRIPTION
**Pull request by leahwicz**
### Description
Before releasing the v1.0.0RC1, we need to bump our schema versions for changes that were made

Related PR for schema bump: https://github.com/dbt-labs/schemas.getdbt.com/pull/6

### Checklist

- [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,13 @@
 ### Features
 - Allow nullable `error_after` in source freshness ([#3874](https://github.com/dbt-labs/dbt-core/issues/3874), [#3955](https://github.com/dbt-labs/dbt-core/pull/3955))
 - Increase performance of graph subset selection ([#4135](https://github.com/dbt-labs/dbt-core/issues/4135),[#4155](https://github.com/dbt-labs/dbt-core/pull/4155))
+
 ### Fixes
 - Changes unit tests using `assertRaisesRegexp` to `assertRaisesRegex`
 
+### Under the hood
+Bump artifact schema versions for 1.0.0
```
Comment (jtcohen6) [inline CHANGELOG.md:11]: Could we include a brief overview of what structurally changed in `manifest`, `run-results`, + `sources`? For easy reference by users, and by me when I go to update docs.getdbt.com

Comment (leahwicz) [reply CHANGELOG.md]: Oh sorry you asked for a brief description and not just what version was bumped...

Comment (jtcohen6) [reply CHANGELOG.md]: yeah! I have a sense it's very small things that changed

Comment (leahwicz) [reply CHANGELOG.md]: There are more changes than would make sense to write down but I listed the big ones. The small ones are some places we changed `int` to `float` but I listed renames and bigger things like that

**Review by jtcohen6 — APPROVED** [review]

**Pull request merged**